rest-api/participant_enums.py

```diff
@@ -62,6 +62,13 @@ class OrderStatus(messages.Enum):
   PROCESSED = 3
   FINALIZED = 4
 
+class BiobankOrderStatus(messages.Enum):
+  """ The status of a biobank order: finalized/amended/cancelled """
+  UNSET = 0
+  FINALIZED = 1
```

Comment:
So we may instead of UNSET = 0 want to have FINALIZED = 0 and get rid of UNSET... we need to make sure that existing orders are marked as FINALIZED.

Reply:
as discussed, default to null, unset is back.